node_modules/oae-core/activity/js/activity.js

```diff
@@ -122,8 +122,9 @@ define(['jquery', 'underscore', 'oae.core', 'activityadapter'], function($, _, o
             infinityScroll = $('.oae-list', $rootel).infiniteScroll(url, {
                 'limit': 10
             }, '#activity-items-template', {
-                'postProcessor': processActivities,
-                'emptyListProcessor': handleEmptyResultList
+                'ariaLive': 'off',  // Notifications will alert screen readers
```

Comment:
Could it work to add a bit of code in the server? If I understand correctly (which may be doubtful :), the server knows whether any particular event will be pushed to the activity stream and/or the notifications stream. For those events that are going to be pushed to both streams, the server could add a flag to the event in the activity stream that, in effect, says "this event is also going to be pushed to the notifications stream."

The UI could then check for that flag on all events in the activity stream. If the flag is present, then it can suppress screen readers using `aria-live="off"` for that event.

I haven't thought through the implications of this approach w.r.t. aggregation, so it may not really work. But at first glance it seems feasible. (Of course, I've never looked at the server code, so I'm making a big assumption that it would be easy for the server to set such a flag.)